Artroom1/Controller/SavedArtViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SavedArtViewController.swift
+//  Artroom1
+//
+//  Created by Taylor Lindsay on 11/20/18.
+//  Copyright © 2018 Taylor Lindsay at Babbel. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SavedArtViewController: UIViewController {
+    
+    var arrayOfSavedArt: [Artwork]?
+    var savedArtImage: UIImageView!
+
+    //Displays carousel of artworks - for now, images only
+    @IBOutlet weak var savedArt: iCarousel?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        savedArt!.type = .invertedCylinder
+        savedArt!.contentMode = .scaleAspectFit
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Start Over", style: .plain, target: self, action: #selector(startOver))
+    }
+    
+    @objc func startOver() {
+        if let navigationController = navigationController {
+            navigationController.popToRootViewController(animated: true)
+        }
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
What was your intention here?

Reply:
I tried some things but ended up deleting them - I'll remove this since I don't need to override it after all.